<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WgtTileSetTerrainSet</class>
 <widget class="QWidget" name="WgtTileSetTerrainSet">
  <property name="modal" stdset="0">
   <bool>false</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>262</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Build Settings</string>
  </property>
  <property name="sizeGripEnabled" stdset="0">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QFrame" name="frmBorder">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Mode</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbTerrainSetMode">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Match Corners and Sides</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Match Corners</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Match Sides</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnUpward">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnDownward">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnDelete">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frmTerrains">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>60</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="lytTerrains"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddTerrain">
           <property name="text">
            <string>Add Terrain</string>
           </property>
           <property name="icon">
            <iconset resource="../../../../data/HyEditor.qrc">
             <normaloff>:/icons16x16/generic-add.png</normaloff>:/icons16x16/generic-add.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>60</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../../../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/edit-delete.png</normaloff>:/icons16x16/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete Terrain Set</string>
   </property>
   <property name="toolTip">
    <string>Removes the Terrain Set from the TileSet</string>
   </property>
  </action>
  <action name="actionUpward">
   <property name="icon">
    <iconset resource="../../../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/generic-up.png</normaloff>:/icons16x16/generic-up.png</iconset>
   </property>
   <property name="text">
    <string>Order Upward</string>
   </property>
   <property name="toolTip">
    <string>Sorts this terrain set upward in the list</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionDownward">
   <property name="icon">
    <iconset resource="../../../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/generic-down.png</normaloff>:/icons16x16/generic-down.png</iconset>
   </property>
   <property name="text">
    <string>Order Doward</string>
   </property>
   <property name="toolTip">
    <string>Sorts this terrain set downward in the list</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../../data/HyEditor.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>accept()</slot>
  <slot>reject()</slot>
 </slots>
</ui>
